### PR TITLE
Update config-setting-deploy.md

### PR DESCRIPTION
**Pull request by mertenar**
@jaimeo adding note around the caution item that isn't explained yet

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/working-with-managed-desktop/config-setting-deploy.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ We’ll show Desktop background picture in these instructions. After you’ve st
 3. In **Deployment status** workspace, select the setting you want to deploy, and then select the staged deployment to deploy.
 4. Select **Deploy** to deploy the change to one of the deployment groups.
 
+> [!NOTE] 
+> The orange caution icon indicates there is a previous group available for deployment as it’s recommended to rollout in order.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/working-with-managed-desktop/config-setting-deploy.md]:
```suggestion
> The orange caution icon indicates there is a previous group available for deployment as it’s recommended to roll out in order. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@mertenar - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@jaimeo - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**